modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In console, select *Operators* -> *Installed Operators*.
+. In the *Provided APIs* column, click *Central* to view the *stackrox-central-services* page, if you installed {product-title-short} using the default values.
+. Click the name of the cluster where Central was installed. The default value is *stackrox-central-services*.
+. Select the YAML tab.
+. Edit the YAML file as shown in the following example:
++
+[source,yaml]
+----
+  scannerV4:
+    db:
+      persistence:
+        persistentVolumeClaim:
+          claimName: scanner-v4-db
+    indexer:
+      scaling:
+        autoScaling: Enabled
+        maxReplicas: 5
+        minReplicas: 2
+        replicas: 3
+    matcher:
+      scaling:
+        autoScaling: Enabled
+        maxReplicas: 5
+        minReplicas: 2
+        replicas: 3
+    scannerComponent: Enabled
```

Comment:
Consider only showing the `scannerComponent` setting here (and in the following section) - since the other values are the defaults - if the defaults were to ever change (ie: replicas or claim name) we may not want users copying what is here into the CRs yaml.

```suggestion
  scannerV4:
    scannerComponent: Enabled
```